test-app/tests/integration/helpers/format-date/api-that-may-be-deprecated-test.ts

```diff
@@ -13,6 +13,7 @@ module(
     test('input is null and allowEmpty is false', async function (assert) {
       await render(hbs`
         <div data-test-output>
+          {{! @glint-expect-error }}
```

Comment:
In commit 3, we see that the tests from #1811 cause TS errors, but Glint wasn't able to catch them. This was partly due to the monorepo setup (the test app is separate from the addon), and maybe partly due to `ember-intl` not being a v2 addon.